RELEASE-NOTES.txt

```diff
@@ -1,9 +1,9 @@
 18.8
 -----
-* [***] Fixes a crasher that was sometimes triggered when seeing the details for like notifications.
-
 * [*] Editor: Show a compact notice when switching between HTML or Visual mode. [https://github.com/wordpress-mobile/WordPress-iOS/pull/17521]
 * [***] Fixed crash where uploading image when offline crashes iOS app. [#17488]
+* [***] Fixed crash that was sometimes triggered when deleting media. [#17559]
+* [***] Fixes a crasher that was sometimes triggered when seeing the details for like notifications. [#17529]
```

Comment:
Fixing a release note entry from this PR: https://github.com/wordpress-mobile/WordPress-iOS/issues/17529